datastore/datastore_test.go

```diff
@@ -1413,6 +1423,22 @@ var testCases = []testCase{
 		"",
 		"",
 	},
+	{
+		"embedded time field",
+		&SpecialTime{MyTime: EmbeddedTime{now}},
+		&SpecialTime{MyTime: EmbeddedTime{now}},
+		"",
+		"",
+	},
+	{
+		"embedded time load",
+		&PropertyList{
+			Property{Name: "MyTime.", Value: now, NoIndex: false, Multiple: false},
```

Comment:
Yea, I argued for that too. But it would break backwards compatibility. 
See discussion here: https://github.com/golang/appengine/issues/41

Unfortunately, this is broken on the cloud.google.com/go/datastore side too, and I'm going to have to do something like "MyTime.Time" on that side. So interop btwn the two packages will be broken for users with these embedded times or geo points.